### PR TITLE
Move default cursor point outside of the page (closes #794)

### LINKED ISSUE #794: Hover action is not executed if the target point coordinates are equal with default cursor position
Now reproduces if you call t.hover() to the page position x=50 y=50

### PR DESCRIPTION
**Pull request by AlexanderMoskovkin**
/cc @helen-dikareva @AndreyBelym @VasilyStrelyaev 

I suppose there are no reason to write a test for this. I can write a test that hovers to point (50;50) but it is not a common case.

### CODE REVIEW

File: src/client/automation/cursor.js [hunk 1]
```diff
@@ -5,9 +5,10 @@ var domUtils = testCafeCore.domUtils;
 var cursorUI = window.top === window ? testCafeUI.cursorUI : testCafeUI.iframeCursorUI;
 
 
+// NOTE: default x,y should be a point outside of the page (GH-794)
```
Comment (VasilyStrelyaev) [inline src/client/automation/cursor.js]:
```
// NOTE: the default position should be outside of the page (GH-794)
```

**Comment by VasilyStrelyaev** [comment]
\r-

**Comment by AlexanderMoskovkin** [comment]
fpr

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by helen-dikareva** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
was an unstable test. /cc @AndreyBelym

**Comment by churkin** [comment]
lgtm

**Pull request merged**